django-verdant/rca/migrations/0054_auto_20170120_1221.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.9 on 2017-01-20 12:21
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('rca', '0053_auto_20170120_1212'),
```

Comment:
Update this to ``0052_auto_20170120_1206`` when ``django-verdant/rca/migrations/0053_auto_20170120_1212.py`` is deleted